### PR TITLE
feat(project.activate): instanciate the discovery project activation page

### PR DESCRIPTION
**Pull request by brofik**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/TAPC-97` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #TAPC-96 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description
The goal of this Pull Request is to instanciate the discovery project activation page  with its responsive layout and connect this page to the two activation banners (project dashboard banner and database banner).

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/activate/activate.controller.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import illustration from './assets/activate-bg.png';
+
+export default class {
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/activate/activate.controller.js:3]:
```suggestion
export default class ProjectActivateController {
```

File: packages/manager/modules/pci/src/projects/project/activate/activate.controller.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  /* @ngInject */
+  constructor($q, orderCart, pciProjectNew) {
+    this.$q = $q;
+    this.orderCart = orderCart;
+    this.pciProjectNew = pciProjectNew;
+    this.illustration = illustration;
+  }
+
+  onActivateProjectClick() {
+    this.goToProjectDashboard();
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/activate/activate.controller.js:13]:
```suggestion
    return this.goToProjectDashboard();
```

File: packages/manager/modules/pci/src/projects/project/activate/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  "pci_projects_project_breadcrumb": "Activation projet",
+  "pci_projects_project_activate_title": "Activer votre projet",
+  "pci_projects_project_activate_description": "La validation d'un moyen de paiment est obligatoire pour activer votre projet. Si vous choisissez de ne consommer aucune ressource Public Cloud, vous ne serez pas débité.",
+  "pci_projects_project_activate_cta": "Activer votre projet",
+
+  "pci_projects_project_pci_banner_promotion_title": "200€ offerts",
+  "pci_projects_project_pci_banner_promotion_description_1": "Valable 1 mois pour tester nos services après l'ouverture de votre projet.",
+  "pci_projects_project_pci_banner_promotion_description_2": "Vous serez débité uniquement si vous poursuivez l'utilisation de nos produits au delà de cette somme ou cette date. De quoi démarrer sereinement.",
+  "pci_projects_project_pci_banner_section_1_title": "Facturation granulaire",
+  "pci_projects_project_pci_banner_section_1_description": "La facturation des ressources se fait en fonction de votre utilisation. Si vous utiliser un produit une heure, vous ne serez facturée que de cette heure là.",
```
Comment (darsene) [inline packages/manager/modules/pci/src/projects/project/activate/translations/Messages_fr_FR.json:11]:
```suggestion
  "pci_projects_project_pci_banner_section_1_description": "La facturation des ressources se fait en fonction de votre utilisation. Si vous utilisez un produit une heure, vous ne serez facturée que de cette heure là.",
```

File: packages/manager/modules/pci/src/projects/project/activate/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  "pci_projects_project_pci_banner_section_2_title": "Facturation simple",
+  "pci_projects_project_pci_banner_section_2_description": "Pas de coûts cachés et de surprise en fin de mois. Notre grille tarifaire est construite pour être compréhensible.",
```
Comment (darsene) [inline packages/manager/modules/pci/src/projects/project/activate/translations/Messages_fr_FR.json:13]:
```suggestion
  "pci_projects_project_pci_banner_section_2_description": "Pas de coût caché et de surprise en fin de mois. Notre grille tarifaire est construite pour être compréhensible.",
```
Ca ne devrait pas être au singulier ?

File: packages/manager/modules/pci/src/projects/project/activate/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{
+  "pci_projects_project_breadcrumb": "Activation projet",
+  "pci_projects_project_activate_title": "Activer votre projet",
+  "pci_projects_project_activate_description": "La validation d'un moyen de paiment est obligatoire pour activer votre projet. Si vous choisissez de ne consommer aucune ressource Public Cloud, vous ne serez pas débité.",
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/activate/translations/Messages_fr_FR.json:4]:
```suggestion
  "pci_projects_project_activate_description": "La validation d'un moyen de paiement est obligatoire pour activer votre projet. Si vous choisissez de ne consommer aucune ressource Public Cloud, vous ne serez pas débité.",
```

**Review by darsene — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/activate/translations/Messages_fr_FR.json (hunk 3 above)
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/activate/translations/Messages_fr_FR.json:11]:
```suggestion
  "pci_projects_project_pci_banner_section_1_description": "La facturation des ressources se fait en fonction de votre utilisation. Si vous utiliser un produit une heure, vous ne serez facturé que de cette heure là.",
```

File: packages/manager/modules/pci/src/projects/project/project.controller.js [hunk 6]
```diff
@@ -64,4 +64,8 @@ export default class ProjectController {
       return [...features, item.feature].filter((feature) => !!feature);
     }, []);
   }
+
+  onActivateProjectClick() {
+    this.goToDiscoveryProjectActivationPage();
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/project.controller.js:69]:
```suggestion
    return this.goToDiscoveryProjectActivationPage();
```

File: packages/manager/modules/pci/src/projects/project/storages/databases/add/add.controller.js [hunk 7]
```diff
@@ -121,6 +121,11 @@ export default class {
     this.messages = this.messageHandler.getMessages();
   }
 
+  onActivateProjectClick() {
+    // waiting for the tracking label
+    this.goToDiscoveryProjectActivationPage();
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/storages/databases/add/add.controller.js:126]:
```suggestion
    return this.goToDiscoveryProjectActivationPage();
```

**Review by BrunoMarquesOVH — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html [hunk 8]
```diff
@@ -36,7 +36,7 @@
         data-translate="pci_database_add_discovery_project_banner_message"
     ></span>
     <!-- The href link will be added when the discovery project activation page is implemented  -->
-    <a href="#">
+    <a data-ng-click="$ctrl.onActivateProjectClick()" rel="noopener">
```
Comment (y4nnL) [inline packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html:39]: **nitpick**: avoid having link elements without a href attribute (accessibility)
Please replace by a button element with the `oui-button_link` class or a `oui-button` component with a `link` variant property

**Review by y4nnL — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html (hunk 8 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html:39]: From what I saw in the UI-Kit, the link variant has a hover effect that don't replicate the hover behaviour of an `<a>` element.
https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-buttons-webcomponent-ghost--default
Which make it an unsuitable choice to fit the feature request.

I've already suggested to the UI designer to use a button for this component for the same argument that you proposed. But the hyperlink option was the one that was selected.

Comment (ghyenne) [reply packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html:39]: if you prefer to stay with `<a>` you can use ui-sref and the data-track-on / data-track-type / data-track-name

Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html]: Thanks @ghyenne 
I've implemented your suggestion.

**Review by ghyenne — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html [hunk 9]
```diff
@@ -36,7 +36,7 @@
         data-translate="pci_database_add_discovery_project_banner_message"
     ></span>
     <!-- The href link will be added when the discovery project activation page is implemented  -->
-    <a href="#">
+    <a ui-sref="pci.projects.project.activate" rel="noopener">
```
Comment (ghyenne) [inline packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html:39]:
```suggestion
    <a data-ui-sref="pci.projects.project.activate" rel="noopener">
```

**Pull request merged**